includes/controls/url.php

```diff
@@ -64,13 +65,15 @@ public function get_default_value() {
 	protected function get_default_settings() {
 		return [
 			'label_block' => true,
-			'show_external' => true,
 			'placeholder' => __( 'Paste URL or type', 'elementor' ),
 			'autocomplete' => true,
+			'options' => [ 'is_external', 'nofollow', 'custom_attributes' ],
 			'dynamic' => [
 				'categories' => [ TagsModule::URL_CATEGORY ],
 				'property' => 'url',
 			],
+			'custom_attributes_description' => __( 'Set custom attributes for the anchor element. Separate attribute keys from values 
+			using the | (pipe) character. Separate key-value pairs with a comma. Do not use spaces.', 'elementor' ),
```

Comment:
Use trimming instead of forcing the user not to insert spaces

Reply:
trimming is ok, but people can put spaces in their keys/values. trimming doesn't solve that problem. I will add trimming, but I do think that part of the description should stay